docker-compose.yml

```diff
@@ -37,5 +38,29 @@ services:
       # DOZZLE_USERNAME=
       # DOZZLE_PASSWORD=
 
+  ocrd-database:
+    image: "mongo:latest"
+    container_name: ocrd-database
```

Comment:
Why the fixed `container_name` here? IIUC it should also work having Docker daemon assign a default name (based on the Docker network, as all other services), but still referencing MongoDB by the service name (i.e. Docker should map service to host names).

Reply:
For me it **does work** without `container_name` (keeping the MongoDB URL as is).

Reply:
Wait. I do get an occasional failure to start mongo-express now (`Could not connect to database using connectionString: mongodb://root:1234@ocrd-database:27017/"`) – despite the `condition: service_started`...

Reply:
see https://github.com/slub/ocrd_monitor/commit/827d9aa0c874bf5ce895d69aa01efeff618c4203